app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) == 3 and parts[0] == "mv":
+        source_path = parts[1]
+        destination_path = parts[2]
+        if "/" in destination_path:
```

Comment:
Do not use any specific directory delimiters in the task, your code should be platform-agnostic